index.js

```diff
@@ -140,7 +148,13 @@ async function run() {
     core.setSecret('ghToken');
     core.setSecret('chToken');
 
-    const { pull_request: pullRequest, repository } = github.context.payload;
+    const octokit = github.getOctokit(ghToken);
+    const { number: prNumber, repository } = github.context.payload;
+    const { data: pullRequest } = await octokit.pulls.get({
+      repo: repository.name,
+      owner: repository.owner.login,
+      pull_number: prNumber,
+    });
```

Comment:
What effect does this have? It looks like it should work the same either way to me. The old way seems simpler.

Reply:
The workflow where I'm using `clubhouse-pr` has two steps, 1) Add some urls to the PR's body, 2) Call `clubhouse-pr`, which also updates the body. 

`github.context.payload.pull_request` contains the PR's state from the beginning of the workflow, so when the 2. step it executed with `clubhouse-pr`, it overrides the changes from the 1. step because it's looking at `github.context.payload.pull_request`, which doesn't contain the changes that were made in the 1. step. I'm therefore retrieving the PR in its latest state in line 162.